mms/tests/integration_tests/onnx_export_integ_test.py

```diff
@@ -34,46 +59,87 @@ def _download_file(download_dir, url):
 
 def cleanup(tmpdir):
     print("Deleting all downloaded resources for SSD MXNet Model Server Integration Test")
-    shutil.rmtree(tmpdir)
+    shutil.rmtree(str(tmpdir))
 
 
-def setup_onnx_integ(tmpdir):
+def setup_onnx_integ(tmpdir,onnx_model,port):
     # Start the mxnet model server for onnx integ tests.
     print("Starting MXNet Model Server for onnx integ test..")
 
     # Set argv parameters
     sys.argv = ['mxnet-model-server']
-    sys.argv.append("--models")
-    sys.argv.append("squeezenet={}/squeezenet.model".format(tmpdir))
+    sys.argv.append('--models')
+    sys.argv.append("{}={}/{}.model".format(onnx_model, tmpdir, onnx_model))
+    sys.argv.append('--port')
+    sys.argv.append(port)
     mxnet_model_server.start_serving()
 
 
-def create_model(tmpdir):
+def create_model(tmpdir,URL,onnx_model, onnx_source_model_zoo = True):
     # Download the files required for onnx integ tests.
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/squeezenet.onnx")
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/synset.txt")
-    # Download input image.
+    download_dir = tmpdir + '/scratch'
+    try:
+        os.mkdir(download_dir)
+        if onnx_source_model_zoo:
+                _download_file(download_dir, URL[onnx_model])
+                _download_file(download_dir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
```

Comment:
We want tests that are generic and scale to new models (think RNNs).
I think it will be better to have dedicated signature and synset per model, rather than reusing squeezenet's.
This will make adding new models trivial and code-updates-free: just drop in new model files into a certain bucket - and the system picks this new model up automatically.

Reply:
@lupesko I have added this in the issue#340